default.css

```diff
@@ -7,45 +7,44 @@
   --clr-light: 40, 38%, 92%;
   --clr-white: 0 0% 100%;
   --clr-red: 338, 81%, 41%;
-  --clr-error: 345, 69%, 49%;
-  --clr-info: 198, 74%, 47%;
-  --clr-warning: 46, 100%, 90%;
-  --clr-success: 153, 57%, 82%;
+  --clr-blue: 198, 74%, 47%;
+  --clr-yellow: 46, 100%, 90%;
+  --clr-green: 153, 57%, 82%;
 
   /* font-sizes */
   --fs-900: clamp(5rem, 8vw + 1rem, 9.375rem);
-  --fs-800: 3.5rem;
+  /* --fs-800: 3.5rem;
   --fs-700: 1.5rem;
   --fs-600: 1.25rem;
   --fs-500: 1rem;
-  --fs-400: 0.9375rem;
+  --fs-400: 0.9375rem; */
+  --fs-800: 4.5rem;
+  --fs-700: 3.5rem;
+  --fs-600: 2rem;
+  --fs-500: 1.75rem;
+  --fs-400: 1.125rem;
   --fs-300: 1rem;
   --fs-200: 0.875rem;
 
   /* font-family */
   --ff-serif-openSans: "Open Sans", sans-serif;
 }
 
-@media (min-width: 35em) {
+@media (max-width: 45em) {
   :root {
-    --fs-800: 5rem;
-    --fs-700: 2.5rem;
-    --fs-600: 1.5rem;
-    --fs-500: 1.25rem;
-    --fs-400: 1rem;
+    --fs-700: 3rem;
   }
 }
 
-@media (min-width: 45em) {
+/* @media (min-width: 45em) {
   :root {
-    /* font-sizes */
     --fs-800: 6.25rem;
     --fs-700: 3.5rem;
     --fs-600: 2rem;
     --fs-500: 1.75rem;
     --fs-400: 1.125rem;
   }
-}
+} */
```

Comment:
remove commented code if it is of no use

Reply:
https://discord.com/channels/914867137863118919/927786435334004787/940298016320028773

Reply:
done